### DISCUSSION TITLE
Possible restoration of unused Butcher movie

### DISCUSSION THREAD

**Discussion by Caesar0007** [diasurgical/DevilutionX#5548] [Ideas]
There is an unused Butcher movie in MPQ. Is it possible to have it played in game?

https://www.youtube.com/watch?v=LeOhX3E7W3Q

**Comment by StephenCWills** [comment]
Sure, it's possible to have it played. However, it's not currently included in DevilutionX. I don't know of any plans to add it in, either.

**Reply by Caesar0007** [reply]
At least for single player as an option.

**Reply by AJenbo** [reply]
When?

**Reply by Caesar0007** [reply]
I'm not a programmer, but played Diablo/Hellfire for a long time. I can only provide suggestions, ideas and testing. When it's can be done - only depends on people, that support this project.

**Reply by DakkJaniels** [reply]
Pretty sure he means when is it supposed to play...

**Reply by qndel** [reply]
Belzebub/Tchernobog make it play when you approach the butcher's chamber

**Reply by Caesar0007** [reply]
Exactly! That's great idea. It can be played in SP at action for door open. Find this little movie 20 years ago in MPQ and was very supprised. There is some additional movies, that was unused if I remember right.

**Reply by AJenbo** [reply]
I guess it would be possible to do it when ever his AI wakes, but keep track of if it has already been played. My guess is that this is where it was originally but it was removed as it caused the game to choke after reading the movie from the CD. So you where probably dead by the time the frame rate returned to normal.

I'm not really thinking we are going to add it, but wouldn't hurt to try it out.

Removed quest: https://www.youtube.com/watch?v=4nENHi_CFkk

**Comment by ghost** [comment]
As a player i am strongly opposed to the cinematics for multiplayer. That cutscene is amazing so it could be cool for singleplayer.

**Reply by Zulgrib** [reply]
If it's an option, is it a problem?

**Reply by AJenbo** [reply]
You probably won't get a reply since they deleted their account (aka. ghost)

**Comment by ghost** [comment]


**Reply by DakkJaniels** [reply]
What is the quest supposed to be? Single Player quests in MP doesn't really involve making up new stuff, so it's not really equivalent. I get there is some unused dialogue, but that's only a small part of making a quest.

**Reply by AJenbo** [reply]
I think it would make more sens as a Lua based mod

**Reply by ghost** [reply]
I don't know the extent of what is included in the original game files as I do not have a basic understanding of the game code. However, if ALL there is, is dialogue (which I don't believe is true), I would agree that this falls more into the additional mods realm. Surely the quest locations are also included?

**Reply by StephenCWills** [reply]
There are no quest locations for the Butcher besides the one you see in the game.

**Reply by AJenbo** [reply]
>  what is included in the original game files

- The Map of the Stars
- A maze map (that is not referenced any where else)
- The dialog you inked
- A few unused spells
- Fire monsters
- Things you see in Hellfire

Everything else in Beezlebub is custom content for that mod
